onair/src/ai_components/planners_interface.py

```diff
@@ -10,22 +10,16 @@
 """
 Planners interface class for managing all planning-based AI components
 """
-import importlib.util
-
+from ..util.plugin_import import import_plugins
 from ..util.data_conversion import *
 
 class PlannersInterface:
-    def __init__(self, headers, _ai_plugins={}):
+    def __init__(self, headers, _planner_plugins={}):
         assert(len(headers)>0), 'Headers are required'
         self.headers = headers
-        self.ai_constructs = []
-        for module_name in list(_ai_plugins.keys()):
-            spec = importlib.util.spec_from_file_location(module_name, _ai_plugins[module_name])
-            module = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(module)
-            self.ai_constructs.append(module.Plugin(module_name,headers))
+        self.ai_constructs = import_plugins(self.headers,_planner_plugins)
```

Comment:
Oh, are they call ai_constructs for both?

Reply:
yes